web/src/components/CityNotCooperatingFooter.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { CITY_NOT_COOPERATING_ROUTE } from 'api-client/src'
+
+import cityNotCooperatingIcon from '../assets/cityNotCooperating.svg'
+import { createPath } from '../routes'
+
+const FooterContainer = styled.div`
+  background-color: ${props => props.theme.colors.backgroundAccentColor};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  border-bottom: 2px solid black;
```

Comment:
I know this is in the designs as well but for me it looks like a glitch, I think dark grey/or no line at all would look better. But again, just personal preference.

However, you should use a color from our theme (there should be black).